src/app/Mappings/components/MappingBuilder/MappingBuilder.tsx

```diff
@@ -20,6 +20,7 @@ export interface IMappingBuilderItem {
 export const mappingBuilderItemsSchema = yup
   .array<IMappingBuilderItem>()
   .required()
+  .min(1)
```

Comment:
Just a detail but this doesn't seem relevant to this PR.

Reply:
After #566 (including a `yup` version upgrade) the handling of required arrays changed, and an empty array became a valid value for a required array field. This meant that without the `min(1)` here you could skip right past the mapping steps in the wizard without filling them in. I figured as part of adding a new wizard step I should make sure the wizard step flow is correct. I'll put it in another PR if you want though.